### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by greg0ire** [comment]
@makasim can you help with the Travis build? You introduced some a deprecation for `getChannel`, but did not migrate code in `src/Backend/AMQPBackend.php`

**Comment by makasim** [comment]
The master shouldn't use the deprecated code.

**Comment by greg0ire** [comment]
This is 3.x

File: .travis/install_test.sh [hunk 1]
```diff
@@ -4,7 +4,12 @@ set -ev
 mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
-wget "https://phar.phpunit.de/phpunit-5.7.phar" --output-document="${HOME}/bin/phpunit"
```
Comment (OskarStark) [inline .travis/install_test.sh]: looks like we did the same here like on the article-bundle @greg0ire

Comment (OskarStark) [reply .travis/install_test.sh]: ah no forget it, my fault here

**Comment by OskarStark** [comment]
@makasim can you support us here?

```
Testing Sonata Notification Bundle Test Suite
................................................................. 65 / 78 ( 83%)
.............                                                     78 / 78 (100%)
Time: 7.51 seconds, Memory: 42.00MB
OK (78 tests, 233 assertions)
Generating code coverage report in Clover XML format ... done
Remaining deprecation notices (2)
The Sonata\NotificationBundle\Backend\AMQPBackendDispatcher::getChannel method is deprecated (since 3.2, will be removed in 4.x): 2x
    1x in AMQPBackendTest::testGetIteratorWithPrefetchCount from Sonata\NotificationBundle\Tests\Backend
    1x in AMQPBackendTest::testGetIteratorWithNoPrefetchCount from Sonata\NotificationBundle\Tests\Backend
make: *** [test] Error 1
```

**Comment by greg0ire** [comment]
> Nothing to do with 3.x branch. It uses deprecated stuff. Master does not rely on it any more

That's my point, 3.x can expose deprecated stuff, but shouldn't use it, people should be able to migrate to "the new way" in 3.x and get rid of the deprecations without upgrading to 4.0 (which they cannot for now)

**Comment by makasim** [comment]
@greg0ire @OskarStark see https://github.com/sonata-project/SonataNotificationBundle/pull/341

**Comment by greg0ire** [comment]
Finally :)

**Pull request merged**